Stratis.Bitcoin/BlockStore/BlockStoreFeature.cs

```diff
@@ -42,7 +42,7 @@ public class BlockStoreFeature : FullNodeFeature
 			this.nodeLifetime = nodeLifetime;
 			this.connectionManager = connectionManager;
 			this.nodeSettings = nodeSettings;
-		    this.storeLogger = loggerFactory.CreateLogger("Stratis.Bitcoin.BlockStore");
+		    this.storeLogger = loggerFactory.CreateLogger(this.GetType().FullName);
```

Comment:
FullName won't give you the namespace but rather the namespace + the type.
As suggested in my review, this needs to be this.GetType().Namespace if it needs to be like before.